modules/virt-supported-cluster-version.adoc

```diff
@@ -6,5 +6,4 @@
 [id="virt-supported-cluster-version_{context}"]
 = {VirtProductName} supported cluster version
 
-{VirtProductName} {VirtVersion} is supported for use on {product-title} {product-version} clusters.
-
+{VirtProductName} {VirtVersion} is supported for use on {product-title} {product-version} clusters. To use the latest z-stream release of {VirtProductName}, you must have updated your cluster to the latest version of {product-title} {product-version}.
```

Comment:
You don't need the {product-version} variable at the end of the second sentence. 

I would rewrite the second sentence as: "To use the latest z-stream release of {VirtProductName}, you must first upgrade to the latest version of {product-title}."